src/test/kotlin/blackjack/domain/game/BlackJackGameTest.kt

```diff
@@ -0,0 +1,96 @@
+package blackjack.domain.game
+
+import blackjack.domain.player.playerNames
+import blackjack.domain.score.CardScoreCalculator
+import blackjack.domain.shuffle.CardNotShuffler
+import blackjack.domain.shuffle.CardShuffler
+import io.kotest.core.spec.style.BehaviorSpec
+import io.kotest.matchers.shouldBe
+
+class BlackJackGameTest : BehaviorSpec({
+
+    Given("카드를 분배하면") {
```

Comment:
```suggestion
    When("카드를 분배하면") {
```
"카드를 분배한다" 라는 것은 행위이기 때문에, given이 아닌 when에 해당합니다.
given-when-then에 대해서 학습해보시면 좋을 것 같아요!

ref. https://brunch.co.kr/@springboot/292

Reply:
반영했습니다 :0
https://github.com/next-step/kotlin-blackjack/pull/465/commits/84264199eea82e01fc7cbc0a9ab00d293cbd3444